exercises/secret-handshake/example.js

```diff
@@ -12,13 +12,13 @@ function SecretHandshake(handshake) {
     return this.shakeWith;
   };
 
-  this.calculateHandshake = function (handshake) {
+  this.calculateHandshake = function (handshakeInner) {
     /* jshint bitwise:false */
```

Comment:
We're not using jshint, so let's remove this.

Reply:
I found this when I opened this file originally. Not sure if it got added on my end (quite possible--I use jshint), but I'll go ahead and remove it.